templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -34,9 +34,8 @@
     "from alpineer import io_utils, load_utils, misc_utils\n",
```

Comment:
<p>The other notebooks use <code>segmentation_dir</code>, any reason to change it here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>